pages/docs/dev-docs/dev-testing.md

```diff
@@ -55,7 +55,7 @@ The latter is only useful for running individual tests in a debugger.
 
 Tests in CTest are named `precice.` followed by the full test name of boost test.
 To get a list of all tests for scripting purposes, use `./testprecice --list_units`.
-CTest additionally groups tests in labels. Use `./testprecice --print-labels` to display existing labels.
+CTest additionally groups tests in labels. Use `./testprecice --list_labels` to display existing labels.
```

Comment:
CTest knows the labels, not testprecice. So the mistake is:

```suggestion
CTest additionally groups tests in labels. Use `ctest --print_labels` to display existing labels.
```

Reply:
OK, this works, but now also executes the tests? I expected that it only prints the labels without running the tests. I must admit I don't understand yet what these labels are.

Reply:
The argument is `--print-labels` with a dash. Fixed it above.

Reply:
Looks good now :+1: